cypress/integration/B_000Student_Parent_spec.js

```diff
@@ -0,0 +1,132 @@
+    });
+
+    it("Enters parent details", () => {
+        // parent first name
+        cy.get(":nth-child(2) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("Paula");
+        // parent last name
+        cy.get(":nth-child(3) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("Wan");
+        // relationship to student
+        cy.get(":nth-child(4) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiSelect-root-339 > .MuiSelect-select-340")
+            .click();
+        cy.get("[data-value=\"Mother\"]")
+            .click();
+        // gender
+        cy.get(":nth-child(5) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiSelect-root-339 > .MuiSelect-select-340")
+            .click();
+        cy.get("[data-value=\"Male\"]")
+            .click();
+        // email
+        cy.get(":nth-child(6) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("pwan007@yahoo.com");
+        // phone number
+        cy.get(":nth-child(7) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("9253418562");
+        // birthday
+        cy.get(".student-align > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click();
+        // 1980
+        cy.get(".MuiToolbar-root-73 > .MuiTypography-subtitle1-19").click();
+        cy.contains("1980").click();
+        cy.contains("Mar").click();
+        // day view
+        cy.get(".MuiTypography-h4-16").click();
+        // 13
+        cy.contains("13").click();
+        cy.contains("OK").click();
+        // address
+        cy.get(":nth-child(9) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("123 Main St");
+        // city
+        cy.get(":nth-child(10) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("Pleasanton");
+        // State
+        cy.get(":nth-child(11) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("CA");
+        // Zipcode
+        cy.get(":nth-child(12) > .MuiGrid-container-141 > .MuiFormControl-root-238 > .MuiInputBase-root-273 > .MuiInputBase-input-283")
+            .click()
+            .type("94566");
+    });
+
+    it("Submits the form", () => {
+        cy.get("[style=\"margin-left: 2%;\"] > div > .MuiButtonBase-root-122")
+            .click();
+        cy.contains("successfully");
```

Comment:
We need to do a thorough check of if this use case has been successfully submitted, so this is just the action to perform the flow, but we need to check if this flow was successful in adding a student and parent that gets reflected across the platform. When we write assertions, we should be using `cy.should()` https://codelikethis.com/lessons/javascript/cypress

We'll also need to write the actual assertion for this test by checking if we see a student and a parent appear when we search for them - to do a thorough test, we need to test to see if the student/parent ids we received can be found through Search and can we access their account pages that reflect the student/parent id's that we received from the original POST request.